guides/hack/11-built-in-types/28-shapes.md

```diff
@@ -84,7 +84,7 @@ function uses_shape_internally(): void {
 }
 ```
 
-For large shapes, it is often convenient to define a type alias.
+For large shapes, it is often convenient to define a type alias.  This is useful because it creates a contract users of your function are expected to follow.
```

Comment:
So does a raw inline definition; there's two major benefits of a type alias:
- having a name for the type that can help thinking about it
- it helps re-use, and making it clear where the same type is being used